correlation.go

```diff
@@ -48,6 +48,9 @@ func PutCorrelation(ctx context.Context, key, value string) bool {
 		return false
 	}
 	correlationContext := span.context().CorrelationContext
+	if correlationContext == nil {
+		return false
+	}
```

Comment:
`#createSegmentContext` has included the initialization. Do you still need this?

Reply:
It's a recheck. I think don't need this. deleted it.